readme.md

```diff
@@ -9,13 +9,34 @@ Using CRDTs to build a collaborative text editor, is mostly a solved problem. Tu
 
 [Repo](https://github.com/chewnoill/collaborative-editor)
 
+## Prerequisites
+
+1. node
+2. docker
+3. postgres
```

Comment:
postgres is installed by docker, is not a direct dependency.

Reply:
Would just listing docker and dbmate be the way to go @chewnoill ?

Reply:
yeah, we only need to list things the developer has to have installed get get started

Reply:
actually I needed to `brew install postgresql` to get dbmate working. I needs `pg_dump`